src/Raven.Server/Indexing/VoronIndexInput.cs

```diff
@@ -41,13 +44,35 @@ private void OpenInternal()
             if (fileTree == null)
                 throw new FileNotFoundException("Could not find index input", _name);
 
-            var readResult = fileTree.Read(VoronIndexOutput.DataKey);
-            if (readResult == null)
-                throw new InvalidDataException($"Could not find data of file {_name}");
+            var numberOfChunks = fileTree.State.NumberOfEntries;
+
+            _basePtrs = new byte*[numberOfChunks];
+            _chunkSizes = new int[numberOfChunks];
+            _streams = new MmapStream[numberOfChunks];
+
+            int index = 0;
+
+            using (var it = fileTree.Iterate(prefetch: true)) // TODO arek - true?
```

Comment:
Probably a bad idea, we are just opening the file here, and we are doing it this way because it is very big, we don't want to force it to read 8GB to memory just because we opened the index. If no one is using it, no point.